src/components/dialogs/NameRequestAuthErrorDialog.vue

```diff
@@ -11,7 +11,7 @@
           <li>Your account is not authorized to access this Incorporation Application &mdash; contact
             the account owner to get access.</li>
           <li>Your login session has timed out &mdash; please exit and then login again.</li>
-          <li>The specified URL is not valid &mdash; exit to return to the Business Registry home page.</li>
+          <li>The specified URL is not valid &mdash; exit to return to the BC Registries home page.</li>
```

Comment:
@severinbeauvais - what page are we taking users back to here?

Reply:
When they click Exit, they go back to the BCROS home page ("Start a Benefit Company and Keep
Cooperatives Records up to date").

Reply:
note to self, take them back to My Business Registry

Reply:
@severinbeauvais '...exit and return to the Business Registry page." (revert to original)

Reply:
Done.